README.md

```diff
@@ -149,6 +149,7 @@ The base model class is a MultiTaskModel. To add another model, first add the cl
 Task-specific components include logistic regression and multi-layer perceptron for classification and regression tasks, and an RNN decoder with attention for sequence transduction tasks.
 To see the full set of available params, see [config/defaults.conf](config/defaults.conf). For a list of options affecting the execution pipeline (which configuration file to use, whether to enable remote logging or tensorboard, etc.), see the arguments section in [main.py](main.py).
 
+<<<<<<< HEAD
```

Comment:
fix

Reply:
Yes, I can do that. I'll talk with Anhad first about ONLSTM and will ask you.